Doc/library/test.rst

```diff
@@ -325,11 +325,6 @@ The :mod:`test.support` module defines the following constants:
    to make writes blocking.
 
 
-.. data:: TEST_SUPPORT_DIR
-
-   Set to the top level directory that contains :mod:`test.support`.
```

Comment:
Why do you remove this one? It still exists, no? If you want to remove the variable itself, I suggest to open a separated issue. TEST_SUPPORT_DIR is not used anywhere in Python.

Reply:
It's used only to compute TEST_HOME_DIR, so I figured it wasn't worth documenting. I can put it back if you prefer.